lib/logzio-nodejs.d.ts

```diff
@@ -0,0 +1,30 @@
+type CallbackFunction = (err: Error, bulk: object) => void;
+
+interface ILoggerOptions {
+    token: string;
+    host?: string;
+    type? : string;
+    sendIntervalMs? : number;
+    bufferSize? : number;
+    debug? : boolean;
+    numberOfRetries? : number;
+    supressErrors?  : boolean;
+    addTimestampWithNanoSecs? : boolean;
+    compress? : boolean;
+    internalLogger : object;
```

Comment:
```suggestion
    internalLogger? : object;
```

Reply:
Also, we need to be explicit in our types.
The `internalLogger` needs a `log` method that receives a string and more args
```suggestion
    internalLogger : { log(message: string, ...args): any } & Record<string, any>;
```
So it would **at least** have the `log` method

Reply:
done